pages/treeherder.py

```diff
@@ -13,9 +14,16 @@
 
 class TreeherderPage(Base):
 
+    _clear_all_menu_locator = (By.CSS_SELECTOR, 'ul.dropdown-menu:nth-child(3) > li:nth-child(4) > a:nth-child(1)')
     _job_details_actionbar_locator = (By.ID, 'job-details-actionbar')
     _job_result_status_locator = (By.CSS_SELECTOR, '#result-status-pane > div:nth-child(1) > span')
     _logviewer_button_locator = (By.ID, 'logviewer-btn')
+    _logviewer_pin_job_locator = (By.CSS_SELECTOR, 'a > span.glyphicon.glyphicon-pushpin')
+    _next_job_title = (By.CSS_SELECTOR, '[class*="btn job-btn btn-dkgray"]')
```

Comment:
This doesn't find anything for me. I suspect it's because of the `.btn-dkgray` which won't always find the next job. If you have an active job, and want to select the next job you could use `.selected-job + .job-btn`, but this won't work if there's no job already selected. In that case you'd want a locator for the first job on the page, something like just `.job-btn`.